mlflow/ml-package-versions.yml

```diff
@@ -49,9 +49,7 @@ tensorflow:
   package_info:
     pip_release: "tensorflow"
     install_dev: |
-      # Unpin `tf-nightly` once the following issue is addressed:
-      # https://github.com/tensorflow/tensorflow/issues/46429
-      pip install 'tf-nightly==2.5.0.dev20210112'
```

Comment:
This PR addresses our handling of early stopping, achieving compatibility with TF 2.5.x, so this pinning can be removed. This particular dev version has also been removed from PyPI.

Reply:
Can we unpin `tensorflow` [here](https://github.com/mlflow/mlflow/pull/4371/files#diff-615abeb3ed96bbeb499868a7ec162c7e7e057d31f490f6b233a3d1b4b8835969R102)?

Reply:
Done! Thanks!